aea/crypto/ethereum.py

```diff
@@ -221,6 +221,12 @@ def send_transaction(
             "gasPrice": self._api.toWei(GAS_PRICE, GAS_ID),
```

Comment:
@Totoual please expose gas_price as a variable as well, both for ethereum and fetchai. The user should be able to specify the gas price they are willing to pay: https://hackernoon.com/ether-purchase-power-df40a38c5a2f